R/misc.R

```diff
@@ -170,3 +170,16 @@ symlog <- function(x) {
 inv_symlog <- function(x) {
   return(sign(x) * (exp(abs(x)) - 1))
 }
+
+#' Logarithm of the sum of exponentials.
+#'
+#' A more numerically stable equivalent to \code{log(sum(exp(x)))}
+#'
+#' @source http://tr.im/hH5A
```

Comment:
The source link seems to be dead

Reply:
OK, will replace with a link to Wikipedia